File: platforms/python/examples/scene/scene/load.py
from fragmentcolor import Scene

# A path — `.gltf` JSON (with external buffers/images) or a `.glb` container.
scene = Scene.load("path/to/model.gltf")

# In-memory `.glb` bytes — fetched from disk, the network, or another
# asset pipeline before this point.
glb_bytes = open("path/to/model.glb", "rb").read()
scene2 = Scene.load(glb_bytes)

File: platforms/web/examples/scene/scene/load.js
import { Scene } from "fragmentcolor";

// Fetch the `.glb` container as bytes, then load it. The same call accepts
// a path string on native; on web pass the bytes instead.
const response = await fetch("/healthcheck/public/model.glb");
const bytes = new Uint8Array(await response.arrayBuffer());
const scene = Scene.load(bytes);